src/json.test.ts

```diff
@@ -54,3 +54,14 @@ test('JSON deep equal', () => {
 
   t({a: 1, b: 2}, {b: 2, a: 1});
 });
+
+test('JSON deepFreeze', async () => {
+  const obj: any = deepFreeze({obj:{}, arr:[]});
```

Comment:
This causes ts warnings that I can't figure out how to silence.

```
  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
  // @ts-ignore
```

... doesn't work.

Reply:
I'll look

Reply:
I figured it out (actually just clicked the little hint from vs code, which figured it out for me).